### PR TITLE
Basic Search Feature

### PR DESCRIPTION
**Pull request by rmr167**
Small pull request whether I implemented a basic search.  My current search finds whether any of the words of the search are included in a question excluding a few stop words such as: a, to, the, etc.

### CODE REVIEW

File: src/main/webapp/scripts/forum-script.js [hunk 1]
```diff
@@ -179,3 +187,37 @@ function collapseReplies(idHandler) {
...
+/**
+ *  Returns whether a word of the search is contained with in the text
+ *
+ *  @param {String} text contents of a question
+ *  @param {String} search contents of the user search
+ *
+ *  @return {Boolean} whether there is a word of the search within the text
+ */
+function containsSearch(text, search) {
+  const stopWords = ['a', 'to', 'and', 'how', 'the', 'when', 'what', 'why',
```
Comment (nihabhaskar) [inline src/main/webapp/scripts/forum-script.js:211]: what is a 'stopWord'

Comment (nihabhaskar) [reply src/main/webapp/scripts/forum-script.js:211]: just for my own understanding

File: src/main/webapp/styles/forum-style.css [hunk 2]
```diff
@@ -146,7 +147,10 @@ i {
 .accepted,
 .accept-button,
 .expand-button,
-.response-form {
+.response-form,
+#search h2,
+#search-input,
+#search-button {
```
Comment (nihabhaskar) [inline src/main/webapp/styles/forum-style.css:153]: could you make the search button the search icon like the magnifying glass

File: src/main/webapp/scripts/forum-script.js (hunk 1 above)
Comment (rmr167) [reply src/main/webapp/scripts/forum-script.js:211]: A stop word is a commonly used word that search engines often ignore https://en.wikipedia.org/wiki/Stop_words

File: src/main/webapp/styles/forum-style.css (hunk 2 above)
Comment (rmr167) [reply src/main/webapp/styles/forum-style.css:153]: made it a search icon :)

**Pull request merged**